File: python/HelloWorld.py
def greet(name):
    """Greet a person by name."""
    return f"Hello, {name}! Welcome to Python."

def main():
    print("=" * 40)
    print("Welcome to Hello World Application")
    print("=" * 40)
    
    name = input("What is your name? ")
    message = greet(name)
    print(message)
    print("=" * 40)

if __name__ == "__main__":
    main()

File: javascript/HelloWorld.js
const readline = require('readline');

function greet(name) {
    return `Hello, ${name}! Welcome to JavaScript.`;
}

function main() {
    const rl = readline.createInterface({
        input: process.stdin,
        output: process.stdout
    });
    
    console.log('========================================');
    console.log('Welcome to Hello World Application');
    console.log('========================================');
    
    rl.question('What is your name? ', (name) => {
        const message = greet(name);
        console.log(message);
        console.log('========================================');
        rl.close();
    });
}

main();
